common/src/autogluon/common/utils/log_utils.py

```diff
@@ -77,9 +77,13 @@ def _check_if_kaggle() -> bool:
 
 
 def _add_stream_handler():
-    stream_handler = logging.StreamHandler()
-    # add stream_handler to AG logger
-    _logger_ag.addHandler(stream_handler)
+    # add stream_handler to AG logger if it doesn't already exist
```

Comment:
Lets add a bit more documentation here since it is called in the root `__init__` files to explain why we do this.